### PR TITLE
Add AST builder test cases for data selection and data projection

### PR DESCRIPTION
**Pull request by sanghee4346**
Goal
--
- [x] Create new test cases for AST Builder Data Selection and Data Projection

Todo
--
- [ ] Write manual

### CODE REVIEW

File: test-suite/src/ast_builder/statements/querying/data_selection_and_projection.rs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        .select()
+        .filter(col("age").gt(30))
+        .project("id, name, age")
+        .execute(glue)
+        .await;
+    let expected = Ok(select!(
+        id | name | age;
+        I64 | Str | I64;
+        4   "Dave".to_owned()   50;
+        5   "Eve".to_owned()   50
```
Comment (devgony) [inline test-suite/src/ast_builder/statements/querying/data_selection_and_projection.rs:44]: You'd better format lines according to this example
```rs
id  | num | name
I64 | I64 | Str;
1     2     "Hello".to_owned();
1     9     "World".to_owned();
3     4     "Great".to_owned();
4     7     "Job".to_owned()
```

**Review by devgony — CHANGES_REQUESTED** [review]

File: test-suite/src/ast_builder/statements/querying/data_selection_and_projection.rs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        .execute(glue)
+        .await;
+    let expected = Ok(select!(
+        id  | age | name;
+        I64 | I64 | Str;
+        1     20    "Alice".to_owned();
+        2     30   "Bob".to_owned();
+        3     30   "Carol".to_owned();
+        4     50   "Dave".to_owned();
+        5     50   "Eve".to_owned()
```
Comment (devgony) [inline test-suite/src/ast_builder/statements/querying/data_selection_and_projection.rs:60]:
```suggestion
        2     30    "Bob".to_owned();
        3     30    "Carol".to_owned();
        4     50    "Dave".to_owned();
        5     50    "Eve".to_owned()
```

**Review by devgony — CHANGES_REQUESTED** [review]

**Review by devgony — APPROVED** [review]
Thank you for contribution 👍🏼

**Review by panarch — APPROVED** [review]
Looks nice! Thanks a lot for the contribution 👍

**Pull request merged**